lte/cloud/go/services/subscriberdb_cache/subscriberdb_cache/main.go

```diff
@@ -32,30 +32,31 @@ func main() {
 		glog.Fatalf("Error creating service: %+v", err)
 	}
 
+	serviceConfig := subscriberdb_cache.MustGetServiceConfig()
+	glog.Infof("Subscriberdb_cache service config %+v", serviceConfig)
+
 	db, err := sqorc.Open(storage.GetSQLDriver(), storage.GetDatabaseSource())
 	if err != nil {
 		glog.Fatalf("Error opening db connection: %+v", err)
 	}
-	digestStore := subscriberdb_storage.NewDigestStore(db, sqorc.GetSqlBuilder())
-	if err := digestStore.Initialize(); err != nil {
-		glog.Fatalf("Error initializing digest storage: %+v", err)
-	}
-
-	fact := blobstore.NewEntStorage(subscriberdb.PerSubDigestTableBlobstore, db, sqorc.GetSqlBuilder())
+	fact := blobstore.NewEntStorage(subscriberdb.SyncstoreBlobstore, db, sqorc.GetSqlBuilder())
```

Comment:
"SyncstoreBlobstoreType" do we use that "Type" suffix elsewhere? Seems like we should

Reply:
I think this is supposed to be the blobstore table name; the lookup table in `subscriberdb` uses `subscriberdb.LookupTableBlobstore`, so I'll change `SyncStoreBlobstore` to `SyncstoreTableBlobstore` for consistency? @hcgatewood

Reply:
👍